### PR TITLE
Autogen network pages (+ local networks & any evm)

### PR DESCRIPTION
**Pull request by DenhamPreen**

### CODE REVIEW

**Comment by DenhamPreen** [comment]
image

Made this quite glaring that there are multiple networks

File: docs/HyperIndex/overview.md [hunk 1]
```diff
@@ -7,34 +7,33 @@ slug: /overview
...
 
-Designed with a core focus on delivering superior performance and a seamless developer experience, HyperIndex optimizes the end-to-end user experience. 
+Designed with a core focus on delivering superior performance and a seamless developer experience, HyperIndex optimizes the end-to-end user experience.
 
 Build a real-time API for your blockchain application in minutes!
 
 ## Key Features
 
 :zap: **[Simplified Setup](/docs/HyperIndex/contract-import.md):** Start instantly with the quickstart, templates, and reference implementations.  
-:zap: **[Any EVM & Fuel](/docs/HyperIndex/supported-networks.md)**: Index smart contract data from any EVM-compatible network.
```
Comment (DenhamPreen) [inline docs/HyperIndex/overview.md:17]: The `.md` is the only change in this file. It's now a folder with an index.md file

File: docs/HyperIndex/supported-networks/any-evm-with-rpc.md
Comment (DenhamPreen) [inline docs/HyperIndex/supported-networks/any-evm-with-rpc.md:1]: This file is manually created

File: docs/HyperIndex/supported-networks/local-anvil.md
Comment (DenhamPreen) [inline docs/HyperIndex/supported-networks/local-anvil.md:1]: This file is manually created

File: docs/HyperIndex/supported-networks/local-hardhat.md
Comment (DenhamPreen) [inline docs/HyperIndex/supported-networks/local-hardhat.md:1]: This file is manually created

File: scripts/update-endpoints.js
Comment (DenhamPreen) [inline scripts/update-endpoints.js:1]: Most of this is auto formatting from prettier

File: scripts/update-endpoints.js [hunk 2]
```diff
@@ -109,41 +137,185 @@ const updateMarkdownFiles = async () => {
...
-      console.log('HyperRPC table not found in the markdown file.');
+      console.log("HyperRPC table not found in the markdown file.");
+    }
+  } catch (error) {
+    console.error("Error updating markdown files:", error);
+  }
+};
+
+// a function to generate a markdown file for each network with a table of endpoints and basic data about the network
+const generateMarkdownFiles = async () => {
```
Comment (DenhamPreen) [inline scripts/update-endpoints.js:189]: Here is the new function to auto generate the markdown networks pages

**Comment by DenhamPreen** [comment]
We should consider rescriptifying or typescriptifying the scripts.

**Comment by moose-code** [comment]
Thanks! Will try review asap 💪

**Review by moose-code — APPROVED** [review]
Looks good 💪 

Think there is always a little room for improvement in the future we can make on some of the descriptions - something too look at in future.

**Pull request merged**